danger-klaxit/lib/danger_plugin.rb

```diff
@@ -59,6 +59,7 @@ def fail_for_bad_commits
   def run_brakeman_scanner
     configs = { app_path: "." }
     configs.merge!(github_repo: github_repo.to_s) if github_repo
+    configs[:rails5] = true if sinatra_project?
```

Comment:
Could you reference the related issue here with a message explaining our will? That would make it easier for someone to refactor it later!

```suggestion
	# Brakeman has a hard time detecting Sinatra projects with ActiveRecord.
	# Hence this trick until a solution is found!
    # See issue_url
    configs[:rails5] ||= true if sinatra_project?
```